pyfar/dsp/dsp.py

```diff
@@ -780,6 +783,12 @@ def regularized_spectrum_inversion(
             numerical aspects of linear inversion. Philadelphia: SIAM, 1998.
 
     """
+    # Deprecation warning for freq_range parameter
+    warnings.warn((
+        'freq_range parameter will be deprecated in pyfar 0.8.0 in favor'
+        ' of frequency_range'),
+            PyfarDeprecationWarning)
```

Comment:
```suggestion
            PyfarDeprecationWarning)
    if frequency_range is None:
        frequency_range = freq_range
```
see below. in this way we just  need to delete this part to remove the old implementation.